deblur/workflow.py

```diff
@@ -427,6 +427,9 @@ def remove_artifacts_seqs(seqs_fp,
         else:
             sim_thresh = 0.65 * 100
 
+    # the minimal average bitscore per nucleotide
+    bitscore_thresh = 0.65
```

Comment:
Overall, I think it is a better idea to expose this as a parameter.  

1. If the user wants to tune this threshold, they can
2. Less maintenance burden for us, so that we decide that we don't like this threshold, we don't have to alter the underlying code.